domains/embeddingbits.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "embeddingbits",
+    "email": "tspamiitesh@gmail.com",
```

```suggestion
    "email": "tspamiitesh@gmail.com"
```
you forgot to remove comma